assets/server/realmadmin/_stats_codes.html

```diff
@@ -17,6 +17,12 @@
       <span class="mr-1">Export as:</span>
       <a href="/stats/realm.csv" class="mr-1">CSV</a>
       <a href="/stats/realm.json" target="_blank">JSON</a>
+      {{if .hasKeyServerStats}}
+      (legacy code data) |
+      <span class="mr-1">Composite Data:</span>
```

Comment:
Since composite _includes_ realm stats, I think this should be an if/else instead of showing both.